src/explore-education-statistics-admin/src/pages/release/edit-release/manage-datablocks/components/ChartAxisConfiguration.tsx

```diff
@@ -54,51 +53,45 @@ const ChartAxisConfiguration = ({
   configuration,
   data,
   meta,
-  labels = {},
+  type,
   capabilities,
-  dataSets = [],
   onChange,
   onSubmit,
 }: Props) => {
-  const sortOptions = useMemo<SelectOption[]>(() => {
-    return [
-      {
-        label: 'Default',
-        value: 'name',
-      },
-      ...Object.values(labels).map<SelectOption>(config => ({
-        label: config.label,
-        value: config.value,
-      })),
-    ];
-  }, [labels]);
+  const dataSetCategories = useMemo<DataSetCategory[]>(() => {
+    if (configuration.type === 'minor') {
+      return [];
+    }
 
-  const limitOptions = useMemo<SelectOption[]>(() => {
-    const configurationWithDataSet: AxisConfiguration = {
+    const config: AxisConfiguration = {
       ...configuration,
       min: 0,
       max: undefined,
-      dataSets,
     };
 
-    const chartData: ChartData[] = createSortedAndMappedDataForAxis(
-      configurationWithDataSet,
-      data.result,
-      meta,
-      labels,
-    );
+    return createDataSetCategories(config, data, meta);
+  }, [configuration, data, meta]);
+
+  // TODO: Figure out how we should sort data
+  // const sortOptions = useMemo<SelectOption[]>(() => {
+  //   return [
+  //     {
+  //       label: 'Default',
+  //       value: 'name',
+  //     },
+  //   ];
+  // }, []);
```

Comment:
Should we get this in Jira with some description, while you're still somewhat in the context of this?

Reply:
Ah I see the ticket... 😅